tests/qos/files/qos_params.th5.yaml

```diff
@@ -0,0 +1,199 @@
+qos_params:
+    th5:
+        topo-t0-standalone-256:
```

Comment:
@rick-arista , I believe we should use topo-t0-standalone-32 here, as it is consistent with our internal setup.

Reply:
These params are generated for the 256 interface config. If you're testing  with 32 interfaces I think they should still work, but you'll need to modify this ad hoc.